plugins/pageembed.md

```diff
@@ -91,6 +91,28 @@ tinymce.init({
 });
 ```
 
+### Inline styles for page embeds
+
+The `tiny_pageembed_inline_styles` setting will inline all styles, instead of using CSS classes, when rendering the embed content. This is useful when the additional CSS classes can't be added to your site. Defaults to `false`.
```

Comment:
technically it keeps the classes as well, in this instance, unlike the other, so maybe "instead of" could be "while also keeping"?

- [ ] consider changing this?

Reply:
done